latte/direction/random.py

```diff
@@ -106,7 +111,5 @@ def find_best_direction(
     budget: int = 200,
     random_generator: Union[int, np.random.Generator] = 1,
```

Comment:
Should we also rename this parameter or no?

Reply:
Good catch! Renamed :slightly_smiling_face: